lld/test/wasm/unresolved-symbols.s

```diff
@@ -1,30 +1,33 @@
-# RUN: llvm-mc -filetype=obj -triple=wasm32-unknown-unknown %s -o %t1.o
+# RUN: split-file %s %t
+# RUN: llvm-mc -filetype=obj -triple=wasm32-unknown-unknown %t/main.s -o %t/main.o
+# RUN: llvm-mc -filetype=obj -triple=wasm32-unknown-unknown %t/secondary.s -o %t/secondary.o
 
-## Check that %t1.o contains undefined symbol undef_func.
-# RUN: not wasm-ld %t1.o -o /dev/null 2>&1 | \
+## Check that %t/main.o contains undefined symbol undef_func.
```

Comment:
`main.o` and `secondary.o` too?

Reply:
Yes, that is what I want to test here.  Without this change the references in `secondard.o` are reported (incorrectly) as coming from `main.o`.

Reply:
I updated the comment to make it more clear what I am testing here.